readme.md

```diff
@@ -42,6 +42,10 @@ gulp.task('default', () =>
 
 #### options
 
+gulp-mocha will pass any options defined directly to the `mocha` binary. That
+means you have every [command line option](http://mochajs.org/#usage) available
```

Comment:
dargs doesn't require that you use only camelCased args, it treats the arguments the same if they're in sausage case (the format of the mocha args) as well.